Application/EdFi.Ods.Sandbox/Provisioners/SqlServerSandboxProvisioner.cs

```diff
@@ -13,16 +13,17 @@
 using EdFi.Admin.DataAccess.Utils;
 using EdFi.Ods.Common.Configuration;
 using log4net;
+using Microsoft.Extensions.Configuration;
 
 namespace EdFi.Ods.Sandbox.Provisioners
 {
     public class SqlServerSandboxProvisioner : SandboxProvisionerBase
     {
         private readonly ILog _logger = LogManager.GetLogger(typeof(SqlServerSandboxProvisioner));
 
-        public SqlServerSandboxProvisioner(IConfigValueProvider configValueProvider,
+        public SqlServerSandboxProvisioner(IConfiguration configuration,
```

Comment:
same as above make an independent copy

Reply:
Creadted new file and wrapped inside NETFRAMEWORK